airbyte-webapp/src/components/EntityTable/components/ConnectionSettingsCell.tsx

```diff
@@ -3,9 +3,9 @@ import styled from "styled-components";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faCog } from "@fortawesome/free-solid-svg-icons";
 
-import useRouter from "hooks/useRouter";
 import { RoutePaths } from "pages/routes";
 import { useCurrentWorkspace } from "hooks/services/useWorkspace";
+import Link from "components/Link";
```

Comment:
I believe this could be imported as 

`import { Link } from "components"` instead